lib/common/theme/theme_sizes.dart

```diff
@@ -0,0 +1,11 @@
+class ThemeSizes {
+  ThemeSizes._();
+
+  // Theme size based on Material Design
+  static const double dimen_14 = 14;
+  static const double dimen_16 = 16;
+  static const double dimen_20 = 20;
+  static const double dimen_24 = 24;
+  static const double dimen_90 = 90;
+  static const double dimen_70 = 70;
```

Comment:
Naming the variable exactly as its value makes little sense. Let's say that tomorrow if we were to update these values e.g. use `18` inside `dimen_14` then naming it with a prefix `_14` would be misleading.

Try using meaningful names for such variables e.g.
```
  static const double tiny = 14;
  static const double small = 16;
  static const double normal = 20;
  static const double large = 24;
  static const double extraLarge = 70;
  static const double huge = 90;
```

Reply:
Tried to copy over the sizes.xml that we had from the traditional android projects, but yeah makes sense
Will fix